### ISSUE TITLE
Decreased circulating apolipoprotein B concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7232]
**Preferred term label:**
Decreased circulating apolipoprotein B concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Reduced circulating level of apolipoprotein B, which is the main apolipoprotein of chylomicrons and low density lipoproteins. It occurs in plasma as two main isoforms, apoB-48 and apoB-100.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
256840   
PMID:16333315   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
